fix warning

geertijewski opened the pull request:
solution by @515orestis https://github.com/minube/geo-timezone/issues/14

Commit:
fix warning

solution by @515orestis https://github.com/minube/geo-timezone/issues/14

nono303 commented:
Hi,
Thanks @geertijewski for this PR!
with php8 `count` on uncountable object trigger an **error** and not a warning 
testing var with `is_countable` before `count` is now mandatory

geertijewski commented:
Hi @nono303 thanks I was not aware of this. However I already removed the invocation of `count()` and replaced it with `is_array()`, so unless I missed something I don't think I need to amend this Pull Request

nono303 commented:
Hi @geertijewski,
you don't need to amend the PR
`is_countable` is nicer before `count` but require >=7.3 https://php.watch/versions/7.3#new-is_countable-function
